File: 1_data/source/wrangle_data.r
# =============================================================================
# Short description of script's purpose
# =============================================================================

library(tidyverse)

# =============================================================================

main <- function() {
  mpg <- read.csv("../input/mpg.csv")
  mpg_clean <- clean_data(mpg)
  save(mpg_clean, file = "../output/mpg.Rdata")
}

clean_data <- function(mpg) {
  # Some data wrangling steps here
  mpg_clean <- mpg 
  return(mpg_clean)
}

# Execute
main()

File: 1_data/source/wrangle_data.py
"""Short description of script's purpose

"""

import pandas as pd
import numpy as np

# =============================================================================

def main():
  mpg = pd.read_csv("../input/mpg.csv")
  mpg_clean = clean_mpg_data(mpg)
  mpg_clean.to_csv("../output/mpg.csv", index = False)

def clean_mpg_data(mpg):
  # Data wrangling steps here
  mpg_clean = mpg 
  return(mpg_clean)

# Execute
if __name__ == "__main__":
	main()